KyleHouston/Problem1.js

```diff
@@ -0,0 +1,179 @@
+var makeCard = // receive factory with external name `makeCard`
```

Comment:
Overall, very nice work! Well designed code throughout. One minor issue: Remember that resource names are case-sensitive. At present, your images aren't loading because `5_of_Hearts.svg` is not the same as `5_of_hearts.svg`. You could change this either by renaming all the files (tedious) or lowercasing the filenames generated by `renderImage` (succinct).